Changelog.md

```diff
@@ -19,6 +19,16 @@
 
   If you encounter such cases, please adjust the type annotation.
 
+* Garbage collection scheduling algorithm is improved once again, and the
+  default garbage collection algorithm is now mark-compact. This change
+  significantly reduces garbage collection overhead in terms of cycles, and
```

Comment:
also in terms of number of gollections, right?

Reply:
Yes, that's what I meant here. In terms of cycles it's actually worse (~30% more cycles), but we do less GC. I'll edit this part.